toonz/sources/translations/german/toonzlib.ts

```diff
@@ -0,0 +1,610 @@
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <source>No restrictions for uncompressed avi video</source>
+        <translation>非圧縮AVIビデオに制限はありません</translation>
+    </message>
+    <message>
+        <source>video width must be a multiple of %1</source>
+        <translation>ビデオ幅は %1 の倍数でなければなりません</translation>
+    </message>
+    <message>
+        <source>video lenght must be a multiple of %1</source>
+        <translation>ビデオの長さは %1 の倍数でなければなりません</translation>
+    </message>
+    <message>
+        <source>No restrictions for this codec</source>
+        <translation>このコーデックに制限はありません</translation>
+    </message>
+    <message>
+        <source>Resolution restrictions:</source>
+        <translation>Beschränkungen für die Auflösung:</translation>
+    </message>
+    <message>
+        <source>It is not possible to communicate with the codec.
+ Probably the codec cannot work correctly.</source>
+        <translation>コーデックの呼び出しができません。
+このコーデックは正常に動作しない恐れがあります。</translation>
+    </message>
+    <message>
+        <source>%1-%2</source>
+        <translation></translation>
+    </message>
+    <message>
+        <source>%1</source>
+        <translation></translation>
+    </message>
+    <message>
+        <source>Expected %1 argument(s) in %2, got %3</source>
+        <translation>Erwartete %1 Argument(e), %3 erhalten</translation>
+    </message>
+    <message>
+        <source>%1 is not a valid color (valid color names are &apos;red&apos;, &apos;transparent&apos;, &apos;#FF8800&apos;, ecc.)</source>
+        <translation>%1 ist keine gültige Farbe (gültige Farbeb sind &apos;rot&apos;, &apos;transparent&apos;, &apos;#FF8800&apos;, usw.)</translation>
+    </message>
+    <message>
+        <source>Vectorization failed</source>
+        <translation>Vektorisierung ist fehlgeschlagen</translation>
+    </message>
+    <message>
+        <source>Argument doesn&apos;t look like a file path : %1</source>
+        <translation>Das Argument ist kein Datei-Pfad: %1</translation>
+    </message>
+    <message>
+        <source>Bad argument (%1): should be an Image (not empty)</source>
+        <translation>Inkorrektes Argument (%1): Es wird ein Bild benötigt</translation>
+    </message>
+    <message>
+        <source>Argument &apos;%1&apos; does not look like a FrameId</source>
+        <translation>Das Argument ist keine Frame-ID</translation>
+    </message>
+    <message>
+        <source>First argument must be a scene : %1</source>
+        <translation>Das erste Argument muss eine Szene sein: %1</translation>
+    </message>
+    <message>
+        <source>Can&apos;t render empty scene</source>
+        <translation>Kann keine leere Szene rendern</translation>
+    </message>
+    <message>
+        <source>New Camera  %1</source>
+        <translation>Neue Kamera  %1</translation>
+    </message>
+    <message>
+        <source>New Pegbar  %1</source>
+        <translation>Neue Pegleiste %1</translation>
```

Comment:
Consider keeping the term Pegbar, as no proper German translation exists (that I know of).